Makefile

```diff
@@ -0,0 +1,19 @@
+SHELL := /bin/bash
+include version.mk
+
+IMAGE_URI=quay.io/redhat/managed-prometheus-exporter-initcontainer
+
+VERSION_MAJOR=0
+VERSION_MINOR=1
+
+all: build tag push
```

Comment:
As these are PHONY targets (there won't be a resulting file matching this name), you need to tag them as such.

This can either be done by adding a line like:
```
.PHONY: all build tag push
```
to mark them all at once, or if you want to prevent merge conflicts, you can mark them separately, closer to their actual rule definition:
```
build:
	docker build . -t $(IMAGE_URI):latest
.PHONY: build
```